### DISCUSSION TITLE
When I turned on virtual environment, oh-my-zsh is showing username and not what zsh shows usually.What i Should do.?

### DISCUSSION THREAD

**Discussion by himanshusuryawanshi** [ohmyzsh/ohmyzsh#11543] [Help] [answered]
![zulipvenv](https://user-images.githubusercontent.com/97045316/222966125-9384fd6c-49b8-452e-8f62-dc484328410e.png)
How do i fix this problem.?

**Comment by mcornella** [comment, marked as answer]
It looks like the activate script resets the prompt variable (`$PROMPT` or `$PS1`).

I can see that this is related to zulip so it's better that you ask over there: https://github.com/zulip/zulip.

Let us know here if you find the answer!
